metadata/society/law_enforcement.md

```diff
@@ -0,0 +1,74 @@
+# Title
+
+Utah Law Enforcement Facilities
+
+## ID
+
+01008931-c755-4df7-aea0-b71aded68f5a
+
+## Brief Summary
+
+Point locations of facilities where sworn officers of a law enforcement agency are regularly stationed.
+
+## Summary
+
+Utah Law Enforcement contains locations where law enforcement officers are regularly based or stationed. Law enforcement agencies may include municipal police, county sheriffs, school police, and federal law enforcement agencies. Additionally, this dataset includes departments within non-law enforcement federal agencies charged with law enforcement (e.g., US Postal Inspectors), and cross jurisdictional authorities (e.g., Port Authority Police).
+
+## Description
+
+### What is the dataset?
+
+The US Bureau of Justice Statistics describes law enforcement agencies as "publicly funded and employ at least one full-time or part-time sworn officer with general arrest powers". This dataset includes point locations for federal, state, local, and special jurisdiction law enforcement agency facilities.
+
+### What is the purpose of the dataset?
+
+This dataset was created to provide a general purpose database of law enforcement facility locations. Law enforcement facility data can been used to asses whether the total police capability of a given area is adequate and to help local governments identify areas of greatest concern. The data are also a helpful resource for situational awareness planning, emergency response, and route analysis.
+
+### What does the dataset represent?
+
+The points in this dataset indicate the locations of law enforcement facilities throughout the state of Utah. Each point includes attributes such as the facility name, street address, and date it was added to the dataset (indicated by the CONTDATE field). Points have been placed on or near the main structure of the facility.
+
+### How was the dataset created?
+
+UGRC derived this dataset from the Homeland Infrastructure Foundation-Level Data (HIFLD) database as maintained by the [US Bureau of Justice Statistics](https://bjs.ojp.gov/). The primary source for this database was the 2008 [Census of State and Local Law Enforcement Agencies](https://bjs.ojp.gov/library/publications/census-state-and-local-law-enforcement-agencies-2008). This dataset is currently maintained by UGRC in collaboration with local and state government authorities.
```

Comment:
```suggestion
UGRC derived this dataset from the Homeland Infrastructure Foundation-Level Data (HIFLD) database maintained by the [US Bureau of Justice Statistics](https://bjs.ojp.gov/). The primary source for this database was the 2008 [Census of State and Local Law Enforcement Agencies](https://bjs.ojp.gov/library/publications/census-state-and-local-law-enforcement-agencies-2008). Our dataset is currently maintained by UGRC in collaboration with local and state government authorities.
```
Replacing "this" with "our" to be a little more clear about which dataset (HIFLD, 2008 census, our layer) we're talking about.